src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/NewAFCalculator.java

```diff
@@ -0,0 +1,165 @@
+package org.broadinstitute.hellbender.tools.walkers.genotyper.afcalc;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.math3.util.MathArrays;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeAlleleCounts;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeLikelihoodCalculator;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeLikelihoodCalculators;
+import org.broadinstitute.hellbender.utils.Dirichlet;
+import org.broadinstitute.hellbender.utils.IndexRange;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public final class NewAFCalculator extends AFCalculator {
+    private static final GenotypeLikelihoodCalculators GL_CALCS = new GenotypeLikelihoodCalculators();
```

Comment:
```GL_CALCS``` is used only to access static methods? in that case just make reference to ```GenotypeLikelihoodCalculators``` directly each time. Could I ask you to make ```GenotypeLikelihoodCalculators``` not instantiable (define a private default constructor and make the class final).